### PR TITLE
Remove unused javascript file

### PR DESCRIPTION
**Pull request by solth**
Remove unused javascript file `metadata.js`.

### CODE REVIEW

File: Kitodo/src/main/resources/messages/errors_de.properties [hunk 1]
```diff
@@ -12,6 +12,7 @@
 error=Fehler...
 
 # A
+anErrorOccurred=Ein Fehler ist aufgetreten\:
```
Comment (henning-gerhardt) [inline Kitodo/src/main/resources/messages/errors_de.properties:15]: I did not found any usage of this key.

File: Kitodo/src/main/resources/messages/errors_en.properties [hunk 2]
```diff
@@ -12,6 +12,7 @@
 error=Error...
 
 # A
+anErrorOccurred=An error occurred\:
```
Comment (henning-gerhardt) [inline Kitodo/src/main/resources/messages/errors_en.properties:15]: see above

File: Kitodo/src/main/resources/messages/errors_es.properties [hunk 3]
```diff
@@ -12,6 +12,7 @@
 error=error...
 
 # A
+anErrorOccurred=Se ha producido un error\:
```
Comment (henning-gerhardt) [inline Kitodo/src/main/resources/messages/errors_es.properties:15]: see above

**Review by henning-gerhardt — CHANGES_REQUESTED** [review]
Looks good but why a new error key was added which is not used is not clear for me.

**Comment by solth** [comment]
@henning-gerhardt thanks for the review. You are right, this message key is not used currently. Actually, I did not _add_ it, though, but just renamed an existing message key to a more appropriate name (from `getNormDataRecordFailed` to `anErrorOccurred`). 

I did not remove it since I was going to use it in another PR, but I will add the key again (with the new name) in that new PR. For now, I have remove the message key as you suggested.

**Review by henning-gerhardt — APPROVED** [review]

**Pull request merged**